jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -75,16 +75,16 @@ pipeline {
             }
             steps {
                 script {
-                    getRepositoryCommit(
-                        componentName: "${COMPONENT_NAME}",
-                        inputManifest: "manifests/${INPUT_MANIFEST}",
-                        outputFile: "commits.yml"
+                    buildInfoYaml(
+                        componentName: COMPONENT_NAME,
+                        inputManifest: "manifests/$INPUT_MANIFEST",
+                        status: "NOT_STARTED",
+                        stage: "START"
```

Comment:
Using `Initialize_stage` better than `START` here, not to be confused with `STARTED/NOT_STARTED`.